packages/gatsby-theme-store/src/sdk/img/fileManager.test.ts

```diff
@@ -0,0 +1,27 @@
+import { optimize } from './fileManager'
+
+test('Image optimization: Return URL with the correct parameters', async () => {
```

Comment:
You can improve the tests descriptions:

```suggestion
describe('Image Optimization', () => {
test('return URL with the correct parameters', () => {
```